portal-web/test/functional/com/liferay/portalweb/macros/WorkflowMetrics.macro

```diff
@@ -118,6 +118,30 @@ definition {
 		Click(locator1 = "WorkflowMetricsSLA#METRICS_SLA_SETTINGS");
 	}
 
+	macro selectDateRange {
+		var key_completionVelocityDateRange = "${selectVelocityDateRange}";
+		var key_startDateRange = "${startDateRange}";
+		var key_endDateRange = "${endDateRange}";
```

Comment:
Hi @joshitagaki , would you please order the variables alphabetically? This is a liferay convention...